keylime/cmd/convert_ima_policy.py

```diff
@@ -180,6 +190,12 @@ def main():
     elif args.ima_policy:
         with open(args.ima_policy, "r", encoding="utf8") as f:
             policy = json.load(f)
+        if not isinstance(policy, dict):
+            print(f"The policy in file {args.ima_policy} must be a dictionary")
+            sys.exit(1)
+    else:
+        policy = {}  # pyright
```

Comment:
Maybe this should be another `copy.deepcopy(ima.EMPTY_IMA_POLICY)` instead? That way we always end up with a valid-looking policy instead of an empty list.

Reply:
Let me actually make this an assert(False). This case cannot happen due to this check here:

```
    if (bool(args.allowlist) and not bool(args.ima_policy)) and (not bool(args.allowlist) and bool(args.ima_policy)):
        print("Either --allowlist or --ima_policy is required!")
        sys.exit(1)
```

Reply:
Yep, I think that was intended when I wrote the code. If there's a more efficient way to evaluate an xor over those two arguments, I welcome it :)

Reply:
I fixed it now... I think, along with adding a case with the deepcopy.

Reply:
> So maybe we do need to force the user to provide either allowlist or ima_policy but not both.

That was my intention when I xor'd `args.allowlist` and `args.ima_policy` as a check.